livekit-rtc/setup.py

```diff
@@ -58,7 +58,12 @@ def finalize_options(self):
     license="Apache-2.0",
     packages=setuptools.find_namespace_packages(include=["livekit.*"]),
     python_requires=">=3.9.0",
-    install_requires=["protobuf>=4.25.0", "types-protobuf>=3", "aiofiles>=24"],
+    install_requires=[
+        "protobuf>=4.25.0",
+        "types-protobuf>=3",
+        "aiofiles>=24",
+        "deprecated>=1.2.18",
```

Comment:
do we want to add a new package just for deprecation?

let's just log a warning no?

Reply:
just saw this comment, either way https://github.com/livekit/python-sdks/pull/395#discussion_r1994960391 ...

Reply:
we can log a warning too! I don't have a preference.. the only preference is making sure users know they are using a deprecated function